core/src/main/java/com/opensymphony/xwork2/ognl/OgnlValueStackFactory.java

```diff
@@ -61,20 +59,17 @@ protected void setTextProvider(TextProvider textProvider) {
         this.textProvider = textProvider;
     }
     
-    @Inject(value="allowStaticMethodAccess", required=false)
-    protected void setAllowStaticMethodAccess(String allowStaticMethodAccess) {
-        this.allowStaticMethodAccess = BooleanUtils.toBoolean(allowStaticMethodAccess);
-    }
-
     public ValueStack createValueStack() {
-        ValueStack stack = new OgnlValueStack(xworkConverter, compoundRootAccessor, textProvider, allowStaticMethodAccess);
+        String allowStaticMethodAccess = container.getInstance(String.class, XWorkConstants.ALLOW_STATIC_METHOD_ACCESS);
+        ValueStack stack = new OgnlValueStack(xworkConverter, compoundRootAccessor, textProvider, Boolean.valueOf(allowStaticMethodAccess));
         container.inject(stack);
         stack.getContext().put(ActionContext.CONTAINER, container);
         return stack;
     }
 
     public ValueStack createValueStack(ValueStack stack) {
-        ValueStack result = new OgnlValueStack(stack, xworkConverter, compoundRootAccessor, allowStaticMethodAccess);
+        String allowStaticMethodAccess = container.getInstance(String.class, XWorkConstants.ALLOW_STATIC_METHOD_ACCESS);
```

Comment:
Hello @lukaszlenart.

Based on an inconsistency found as part of PR #320, it appears that  `XWorkConstants.ALLOW_STATIC_METHOD_ACCESS` should be replaced with `StrutsConstants.STRUTS_ALLOW_STATIC_METHOD_ACCESS` (as STRUTS_ALLOW_STATIC_METHOD_ACCESS is the value used elsewhere in 2.6.x for the static access control flag).

Reply:
But this change targets 2.5.x, those constants were dropped in 2.6 only (`master` branch)

Reply:
Hello @lukaszlenart .

Ahhh .... sorry ... I totally missed that (jumping around between the two branches for some reason I had assumed this PR was referring to 2.6 as opposed to 2.5.x).  If/when this change moves to the 2.6 branch the updated constant would naturally be used.

So please ignore my previous comment-in-error.  :)